shared/modules/utility_module.go

```diff
@@ -0,0 +1,20 @@
+package modules
+
+type UnstakingActor interface {
+	GetAddress() []byte
+	GetStakeAmount() string
+	GetOutputAddress() []byte
+}
+
+type UtilityContext interface {
+	ReleaseContext()
+	GetPersistanceContext() PersistenceContext
```

Comment:
TODO: There's got to be a better way to structure this; shouldn't block this PR, but note to self

Reply:
Added TODO in the code.